src/SamplesApp/SamplesApp.UnitTests.Shared/Controls/UITests/Presentation/SampleChooserViewModel.cs

```diff
@@ -254,124 +254,148 @@ internal async Task RecordAllTests(CancellationToken ct, string screenShotPath =
+					"First test to run cannot be skipped"
+				);
+
+				var tests = testQuery
+					.SkipWhile(testInfo => _firstTargetToRun.HasValue() && !testInfo.Matches(_firstTargetToRun))
+					.Where(testInfo => _targetsToSkip.None(testInfo.Matches))
+					.ToArray();
+
+				if (this.Log().IsEnabled(LogLevel.Debug))
+				{
+					this.Log().Debug($"Generating tests for {tests.Count()} test in {folderName}");
+				}
 
-					if (this.Log().IsEnabled(LogLevel.Debug))
+				foreach (var sample in tests)
+				{
+					try
 					{
-						this.Log().Debug($"Generating tests for {tests.Count()} test in {folderName}");
-					}
+#if XAMARIN
+						var inactiveStats = Uno.UI.DataBinding.BinderReferenceHolder.GetInactiveViewReferencesStats();
+						var activeStats = Uno.UI.DataBinding.BinderReferenceHolder.GetReferenceStats();
+#endif
+
+						var fileName = $"{sample.Category.Category}-{sample.Sample.ControlName}.png";
 
-					foreach (var sample in tests)
-					{
 						try
 						{
-#if XAMARIN
-							var inactiveStats = Uno.UI.DataBinding.BinderReferenceHolder.GetInactiveViewReferencesStats();
-							var activeStats = Uno.UI.DataBinding.BinderReferenceHolder.GetReferenceStats();
-#endif
+							Console.WriteLine($"Creating control for {fileName}");
 
-							var fileName = $"{sample.Category.Category}-{sample.Sample.ControlName}.png";
+							LogMemoryStatistics();
 
-							try
+							if (this.Log().IsEnabled(LogLevel.Debug))
 							{
-								LogMemoryStatistics();
+								this.Log().Debug($"Generating {folderName}\\{fileName}");
+							}
 
-								if (this.Log().IsEnabled(LogLevel.Debug))
-								{
-									this.Log().Debug($"Generating {folderName}\\{fileName}");
-								}
+							await ShowNewSection(ct, Section.SamplesContent);
 
-								await ShowNewSection(ct, Section.SamplesContent);
+							SelectedLibrarySample = sample.Sample;
 
-								SelectedLibrarySample = sample.Sample;
+							Console.WriteLine($"Updating content for {fileName}");
 
-								var content = await UpdateContent(ct, sample.Sample) as FrameworkElement;
+							var content = await UpdateContent(ct, sample.Sample) as FrameworkElement;
 
-								ContentPhone = content;
+							Console.WriteLine($"Assigning ContentPhone {fileName}");
+							ContentPhone = content;
 
-								await Task.Delay(500, ct);
+							Console.WriteLine($"Delaying for {fileName}");
 
-								await GenerateBitmap(ct, folderName, fileName, content);
-							}
-							catch (Exception e)
-							{
-								this.Log().Error($"Failed to execute test for {fileName}", e);
-							}
+							await Task.Delay(500, ct);
 
-#if XAMARIN
-							Uno.UI.DataBinding.BinderReferenceHolder.LogInactiveViewReferencesStatsDiff(inactiveStats);
-							Uno.UI.DataBinding.BinderReferenceHolder.LogActiveViewReferencesStatsDiff(activeStats);
-#endif
+							Console.WriteLine($"Generating screenshot for {fileName}");
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6226524)